### ISSUE TITLE
missing embedded data on root and db resources

### ISSUE THREAD

**Issue by ujibang** [SoftInstigate/angular-restheart#1]
looking at the code:

```
if (angular.isDefined(data._embedded['rh:doc'])) {
                        extractedData = data._embedded['rh:doc'];
                    } else if (angular.isDefined(data._embedded['rh:file'])) {
                        extractedData = data._embedded['rh:file'];
                    } else {
                        extractedData = [];
                    }

```

the embedded data in not actually included for root (that has __embedded['rh:db']_ array) and db (that can have __embedded['rh:bucket']_ and __embedded['rh:coll']_ arrays)

**Comment by ujibang** [comment]
There is a problem with buckets and collection. A db can contain both collections and buckets. So you need to get the two arrays and merge them (for instance, using lodash) eventually.

Also the embedded resources of schema stores (URI /db/_schemas) are rh:schema

**Comment by 22Viig** [comment]
Fixed with commit 1b759dfa22f20b5ebe2954b6b9bb16f6126ce216
